src/Content/Translations.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Kirby\Content;
+
+use Kirby\Cms\Collection;
+use Kirby\Cms\Language;
+use Kirby\Cms\Languages;
+use Kirby\Cms\ModelWithContent;
+
+/**
+ * @package   Kirby Content
+ * @author    Bastian Allgeier <bastian@getkirby.com>
+ * @link      https://getkirby.com
+ * @copyright Bastian Allgeier
+ * @license   https://getkirby.com/license
+ *
+ * @extends \Kirby\Cms\Collection<\Kirby\Content\Translation>
+ */
+class Translations extends Collection
+{
+	/**
+	 * Creates a new Translations collection from
+	 * an array of translations properties. This is
+	 * used in LabPage::setTranslations to properly
+	 * normalize an array definition.
+	 */
+	public static function create(
```

Comment:
`Question`: How is this method (and `Translation::create()` intended to be used? I think the current arguments don't fit our plans to make `Version::create()` a static method. One cannot pass a version instance if it doesn't exist yet.

Reply:
They are indeed only useful this way as long as we still have the setup with Version instances that may not exist yet. We discussed this yesterday. To convert everything to `ModelWithContent::version(): Version|null`  is quite a step at the moment and I'm not fully able to complete/plan it yet. 

I think we should leave it out of this PR and go with the flow here, as long as we agree that those two methods are going to get refactored again and only help temporarily to clean things up.

Reply:
Sure. Could you add `@todo` comments to the code as well?

Reply:
✅